sentry-android-replay/src/test/java/io/sentry/android/replay/ReplayIntegrationWithRecorderTest.kt

```diff
@@ -123,12 +125,22 @@ class ReplayIntegrationWithRecorderTest {
             }
         }
 
-        replay = fixture.getSut(context, recorder, recorderConfig, dateProvider)
+        replay = fixture.getSut(context, recorder, dateProvider)
         replay.register(fixture.scopes, fixture.options)
 
         assertEquals(INITALIZED, recorder.state)
 
+        // have to access 'replayCacheDir' after calling replay.start(), BUT can already be accessed
```

Comment:
hm, so this comment is not relevant anymore?

Reply:
Ah true, let me change it back to the original order then